get_trust_anchor.py

```diff
@@ -80,15 +73,6 @@
 -----END CERTIFICATE-----
 '''
 
-URL_ROOT_ANCHORS = "https://data.iana.org/root-anchors/root-anchors.xml"
```

Comment:
Constants defined in main() are not visible to other functions, they need to be global in order to be used. This includes the NowString.